concordance/win/make_installer.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
```

Comment:
This seems like one of the few times a shell script might be simpler, but I have no objections to this.

Reply:
Yeah, I was on the fence about it.  I like Python's `tempfile.TemporaryDirectory` with a context manager though, so I don't have to worry about cleaning it up.